runtime/lua/vim/loader.lua

```diff
@@ -170,20 +192,6 @@ function Loader.loader_lib(modname)
   return '\ncache_loader_lib: module ' .. modname .. ' not found'
 end
 
---- `loadfile` using the cache
----@param filename? string
----@param mode? "b"|"t"|"bt"
----@param env? table
----@param hash? CacheHash
----@return function?, string?  error_message
----@private
--- luacheck: ignore 312
-function Loader.loadfile(filename, mode, env, hash)
```

Comment:
Removing the entry point for the new global `loadfile` will also mess up the statistics.

Doing `require("whatever")` would now trigger stats for a regular load and additionally for loadfile, so not ideal.

Reply:
Ok, I've just reverted this.